app/scripts/directives/blocks.js

```diff
@@ -251,6 +251,13 @@ angular
               return campusNames.data;
             });
         };
+        if ($scope.answer.value) {
+          $scope.searchCampuses($scope.answer.value).then((data) => {
+            if (data.length < 1) {
```

Comment:
Do you think this conditional could be clearer as `data.length === 0`?

Reply:
yes I'll change that